<launch>

	<node pkg = "cmd_vel_mux" type = "cmd_vel_mux_node" name = "cmd_vel_mux" output = "screen">
	
	<param name = "ackermann_cmd_vel_topic "    type = "string"      value = "/ackermann_steering_controller/cmd_vel"/>
	<param name = "type_cmd_vel_topic"    		type = "string"      value = "/type_cmd_vel"/>

	<param name = "sonar_cmd_vel_topic" 		type = "string"      value = "/cmd_vel/sonar"/>
	<param name = "yaw_cmd_vel_topic" 			type = "string"      value = "/cmd_vel/yaw"/>
	<param name = "line_cmd_vel_topic" 			type = "string"      value = "/cmd_vel/line"/>

	</node>
	
</launch>
